git/00.setup.md

```diff
@@ -2,9 +2,14 @@
 
 ## 0. Setting up git
 
-If git is not installed it can be install via homebrew or by downloading the XCode command line tools.  The homebrew installation is listed below.
+If git is not installed it can be install via homebrew or by downloading the
```

Comment:
Normally I'd suggest not wrapping lines.  An editor can do that wrap and by adding it manually may introduce additional line changes when updating text.